docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,252 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
 
+.. index:: pair: command groups;statistics command group
+
+.. _`Statistics command group`:
+
+``statistics`` command group
+----------------------------
+
+The ``statistics`` command group includes commands that manage and display
+statistics on the WBEM server operations executed by pywbemcli.
+
+.. index:: pair: Operation statistics;statistics commands
+
+.. _Operation statistics:
+
+Statistics  on WBEM server operations can be gathered by the pywbemcli client,
+and also separately by some WBEM servers.  There are multiple components to
+statistics gathering and reporting in pywbemcli:
+
+1. Pywbemcli client gathers statistics when statistics gathering
+is enabled with the ``--timestats`` / ``-T`` general option. The statistics
+gathered includes at least:
+
+a. Number of operations executed by operation type.
+b. Number of exceptions for each operation type.
+c. Operation execution time (minimum, maximum, and average).
+d. Request/response size (minimum, maximum, and average).
+e. Operation name.
+
+In interactive mode, statistics are displayed either after each command (`` -T
+auto) or only only when the command ``statistics show`` is issued (``-T cmd``).
+Once statistics are enabled in the interactive mode they remain enabled and
+only the display mode (cmd/auto) can be modified with the -T general option within
+that interactive session unless the current server is changed with the
+``connection select`` command.
+
+In command line mode, use ``--timestats auto`` to assure statistics are
+displayed since the command mode only displays statistics when specifically
+requested.
+
+When python_mock is used, statistics on compile time of classes
+and instances inserted into the mock respository is also gathered.
+
+2. WBEM servers may support two capabilities for managing statistics on WBEM
+operations:
+
+    a. If statistics gathering is enabled on the WBEM server, each CIM/XML
+       response may include an extra header field ``WBEMServerResponseTime`` with
+       the server response time for that operation. Those response times are
+       gathered in the statistics kept by pywbemcli. This may have to be enabled by
+       setting a statistics gathering flag in the WBEM server.
+
+    b. Some WBEM servers gather statistical information on operation execution
+       time in the WBEM server and report this through instances of a class defined
+       for the WBEM server. This statistics gathering capability is completely
+       independent of pywbemcli. This capability may have to be enabled by setting
+       a statistics gathering flag in the WBEM server. Pywbemcli provides the
+       ``statistics server-show` command to retrieve and organized this data into a
```

Comment:
Grammar: "organized".

Reply:
Removed the organized.  That is our iissue to clean up the data returned from the server.
DONE